src/constructs/acm/certificate.test.ts

```diff
@@ -52,4 +52,33 @@ describe("The GuCertificate class", () => {
     const json = SynthUtils.toCloudFormation(stack) as SynthedStack;
     expect(Object.keys(json.Resources)).toContain("MyCloudFormedCertificate");
   });
+
+  it("should not create a CloudFormation Mapping when used in a GuStackForInfrastructure", () => {
+    const stack = simpleGuStackForTesting();
+    new GuCertificate(stack, {
+      app: "testing",
+      [Stage.CODE]: {
+        domainName: "code-guardian.com",
+        hostedZoneId: "id123",
+      },
+      [Stage.PROD]: {
+        domainName: "prod-guardian.com",
+        hostedZoneId: "id124",
+      },
+    });
+
+    const json = SynthUtils.toCloudFormation(stack) as SynthedStack;
+    expect(json.Mappings).toBeDefined();
```

Comment:
I would probably drop this assertion (and related code) as it should already be covered by earlier snapshot tests

Reply:
Good shout! Done.